### PR TITLE
0.8-Assets.mysql script bug fix

### PR DESCRIPTION
**Pull request by jkokh**
(cherry picked from commit c148024)

### CODE REVIEW

File: platform/plugins/Assets/scripts/Assets/0.8-Assets.mysql [hunk 1]
```diff
@@ -11,11 +11,11 @@ ALTER TABLE `{$prefix}earned`
...
 CHANGE badge_name badgeName varchar(255) NOT NULL,
+ADD COLUMN communityId varbinary(31) NULL DEFAULT NULL AFTER appId,
 CHANGE insertedTime earnedTime timestamp NOT NULL DEFAULT CURRENT_TIMESTAMP AFTER communityId,
 DROP COLUMN associated_id,
-ADD COLUMN communityId varbinary(31) NULL DEFAULT NULL AFTER appId,
 ADD COLUMN publisherId varbinary(31) NULL DEFAULT NULL AFTER badgeName,
 ADD COLUMN streamName varbinary(255) NULL DEFAULT NULL AFTER publisherId,
 DROP INDEX userId,
-ADD INDEX byTime (appId, communityId, earnedTime)
+ADD INDEX byTime (appId, communityId, earnedTime),
```
Comment (EGreg) [inline platform/plugins/Assets/scripts/Assets/0.8-Assets.mysql:20]: this is the only real change?

File: platform/plugins/Assets/scripts/Assets/0.8-Assets.mysql [hunk 2]
```diff
@@ -2,7 +2,7 @@ ALTER TABLE `{$prefix}badge`
 CHANGE app appId varbinary(31) NOT NULL FIRST,
 ADD COLUMN communityId varbinary(31) NULL DEFAULT NULL AFTER appId,
 CHANGE title title varchar(255) NOT NULL,
-ADD COLUMN description TEXT NOT NULL DEFAULT "" AFTER title,
```
Comment (EGreg) [inline platform/plugins/Assets/scripts/Assets/0.8-Assets.mysql:5]: what is the reason for this change, why can't description be empty?

File: platform/plugins/Assets/scripts/Assets/0.8-Assets.mysql [hunk 3]
```diff
@@ -11,11 +11,11 @@ ALTER TABLE `{$prefix}earned`
 CHANGE app appId varbinary(31) NOT NULL FIRST,
 CHANGE userId userId varchar(31) NOT NULL ,
 CHANGE badge_name badgeName varchar(255) NOT NULL,
+ADD COLUMN communityId varbinary(31) NULL DEFAULT NULL AFTER appId,
```
Comment (EGreg) [inline platform/plugins/Assets/scripts/Assets/0.8-Assets.mysql:14]: why did you move this line? just curious. It seems better right before ADD publisherId and all other ADD

File: platform/plugins/Assets/scripts/Assets/0.8-Assets.mysql (hunk 2 above)
Comment (jkokh) [reply platform/plugins/Assets/scripts/Assets/0.8-Assets.mysql:5]: Mysql said me that field of type TEXT can not be declared with default value

File: platform/plugins/Assets/scripts/Assets/0.8-Assets.mysql (hunk 1 above)
Comment (jkokh) [reply platform/plugins/Assets/scripts/Assets/0.8-Assets.mysql:20]: Because comma is required. There will be an SQL error without it.
